### DISCUSSION TITLE
JDBC transaction management without Spring

### DISCUSSION THREAD

**Discussion by chrisparton1991** [micronaut-projects/micronaut-core#6687] [Q&A]
As per https://github.com/micronaut-projects/micronaut-core/issues/6210#issuecomment-927667458, it sounds possible to make use of `@Transactional` (or similar) without Spring, but I can't find anything in the documentation.

Is there an example of this somewhere? For context, I'm targeting Micronaut 3.2.3 + JDBI + SQL Server.

Thanks!

**Comment by graemerocher** [comment]
Just use launch which is already setup with Micronaut tx management https://micronaut.io/launch?type=DEFAULT&name=demo&package=com.example&javaVersion=JDK_11&lang=JAVA&build=GRADLE&test=JUNIT&features=data-jdbc&version=3.2.3&activity=preview&showing=README.md
